frontend/src/components/AccountSentTable.vue

```diff
@@ -1,97 +1,127 @@
 <template>
-  <q-table
-    :grid="$q.screen.xs"
-    card-container-class="col q-col-gutter-md"
-    :columns="mainTableColumns"
-    :rows="formattedSendMetadata"
-    :no-data-label="$t('AccountSendTable.account-empty')"
-    :pagination="paginationConfig"
-    row-key="dateSentUnix"
-    :title="$t('AccountSendTable.sent-funds')"
-  >
-    <template v-slot:top-left="props">
-      <div class="flex">
-        <div class="q-table__title" :props="props">{{ $t('AccountSendTable.sent-funds') }}</div>
-        <base-tooltip class="q-ml-sm self-center" size="xs" icon="fas fa-question-circle">
-          {{ $t('AccountSendTable.storage-description') }}
-        </base-tooltip>
-      </div>
-    </template>
-    <!-- Card Layout for grid option -->
-    <template v-slot:item="props">
-      <div :key="props.row.dateSentUnix" class="col-12">
-        <q-card class="card-border cursor-pointer q-pa-md col justify-center items-center">
-          <q-card-section class="row justify-center items-center">
-            <img class="q-mr-md" :src="props.row.tokenLogo" style="width: 1.2rem" />
-            <div class="text-primary text-h6 header-black q-pb-none">
-              {{ props.row.amount }} {{ props.row.tokenSymbol }}
-            </div>
-          </q-card-section>
-          <div class="row justify-between items-center">
-            <div>
-              <span class="q-mr-xs">{{ $t('AccountSendTable.receiver') }}</span>
-            </div>
-            <div>
-              <div @click="copyAddress(props.row.address, provider)" class="cursor-pointer copy-icon-parent">
-                <span>{{ props.row.addressShortened }}</span>
-                <q-icon color="primary" class="q-ml-sm" name="far fa-copy" />
+  <div>
+    <div class="text-caption q-mb-sm">
+      {{ $t('AccountSentTable.stored-on-device') }}.
+      <router-link
```

Comment:
_(arbitrary comment location to start a thread)_

Maybe we should vertically center align the icons with the text and give them some more padding on the left? 

Also when I hover over the icons I get a cursor and tooltip, but clicking the icons copies the receipient address. Clicking those icons should probably do nothing

<img width="864" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/41400fdc-fef5-4d07-bb5f-34c1553df73e">

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/494/commits/390fa5079818b94afb8194034b6e09fafee2621c I moved the copy icon in front of the address. It seemed like the easiest way to implement things, and we want to change it further we should make a ticket and handle post launch

Reply:
We should handle supporting a different cursor on tooltips in another ticket as it seems like it is tied closely to the base component.

Reply:
Maybe we just drop 390fa50 for now then and fix it post-launch? It looks weird now IMO since they aren't aligned as shown below

<img width="208" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/c24c89a0-20be-41e7-8cf3-a89548396588">

Reply:
https://github.com/ScopeLift/umbra-protocol/issues/515